packages/lesswrong/components/votes/ThreeAxisEmojisVoteOnComment.tsx

```diff
@@ -276,23 +323,36 @@ const ThreeAxisEmojisVoteOnComment = ({
     collection.options.collectionName,
     votingSystem,
   );
-  const {TwoAxisVoteOnComment} = Components;
+  const {OverallVoteAxis, TwoAxisVoteOnComment} = Components;
   return (
-    <div className={classes.root}>
-      <TwoAxisVoteOnComment
-        document={document}
-        hideKarma={hideKarma}
-        collection={collection}
-        votingSystem={getVotingSystemByName("twoAxis")}
-      />
-      {userHasEAEmojiReacts(null) &&
-        <EmojiReactsSection
-          document={document}
-          voteProps={voteProps}
-          classes={classes}
-        />
+    <>
+      {userHasEAEmojiReacts(null)
+        ? (
+          <>
+            <OverallVoteAxis
+              document={document}
+              hideKarma={hideKarma}
+              voteProps={voteProps}
+              showBox
+            />
+            <EmojiReactsSection
+              document={document}
+              voteProps={voteProps}
+              classes={classes}
+            />
+          </>
+        )
+        : (
+          <TwoAxisVoteOnComment
+            document={document}
+            hideKarma={hideKarma}
+            voteProps={voteProps}
+            collection={collection}
+            votingSystem={votingSystem}
+          />
+        )
```

Comment:
Is the plan to test this on the beta site first? I'm a little worried about having two different voting systems on the same post (one on prod and one on beta). I mean I think it will function fine (though wouldn't be that surprised if this causes bugs), but the agree votes would get split between them so some data would be "lost".